mqtt/mqtt-bridge/src/settings.rs

```diff
@@ -1,23 +1,20 @@
 #![allow(dead_code)] // TODO remove when ready
```

Comment:
Do we still need this?

Reply:
yes, if I remove it complains about functions which are not used (like remotes, messages, etc.)